ci/properties/jscrambler-code-integrity.properties.json

```diff
@@ -0,0 +1,7 @@
+{
+    "name": "Jscrambler Code Integrity",
+    "description": "Protect your JavaScript Application with polymorphic obfuscation, code locks, and self-defensive techniques",
+    "creator": "Jscrambler",
+    "iconName": "jscrambler",
+    "categories": ["Continuous integration", "JavaScript"]
```

Comment:
Does code-scanning feel like a better category?

Reply:
Not really! The Code Integrity Github action applies some transformations to the user's source code. Thus calling it Code Scanning can be misleading.

Reply:
Security is also a good candidate for this starter workflow. Any feedback on this matter?

Reply:
We currently keep security templates like code-scanning and dependency review in `code-scanning` directory. Does "Code Scanning" category fit well under security template?

Reply:
Are you appreciate? Is it everything you guys are doing a no for all my heart you're doing the great work and great day and I appreciate it so much that you guys are helping me a lot. Also wanted to give my information iPhone that I have if I can send the pictures or video. I think they have my break jail for Miami. I don't know but I continue hacking my phone it is it's crazy.![image](https://user-images.githubusercontent.com/41128366/213035517-c5d529b2-70c5-4dd5-9244-ca2c5ae2bf2f.png)![image](https://user-images.githubusercontent.com/41128366/213035534-543ea34e-e303-47c3-b799-1b4e1fdc12e2.png)![image](https://user-images.githubusercontent.com/41128366/213035541-c1c462e0-0b36-4e0a-9399-ba2aba4ca78f.png)

Reply:
Moved to `code-scanning`. Thanks